### ISSUE TITLE
Unsubscribe

### ISSUE THREAD

**Issue by felix-lambert** [dis-moi/mobile_poc#143]

**Comment by felix-lambert** [comment]
Can we be more specific with the issue? Is there some UX design for this?

**Comment by prk-dismoi** [comment]
Good point, we have no design for relayed contributions. Will get on this now. @felix-lambert do you handle them right now, i.e. do you fetch and display relayed contributions in the latest APK?

**Comment by JalilArfaoui** [comment]
@felix-lambert are you asking for UX about general unsubscription or how to show that a specific notice is relayed ?

**Comment by felix-lambert** [comment]
I think I need a ux for general unsubscription of a user if I understand well

**Comment by prk-dismoi** [comment]
No UX required on this one, this is a contribution handling issue. The idea is that even if you unsubscribe from contributor 1 (via the app, the way we do it now - no change there), any bubble from contributor 2 (to which a user is subscribed) that relays a bubble from contributor 1 should still be visible.

**Comment by prk-dismoi** [comment]
on that topic @felix-lambert @JalilArfaoui see my question above, do we show relayed bubbles right now?

**Comment by JalilArfaoui** [comment]
Like this 

![image](https://user-images.githubusercontent.com/3037833/125589371-a7926b71-9fc9-40c5-a672-ff577200c0b3.png)

**Comment by JalilArfaoui** [comment]
> I think I need a ux for general unsubscription of a user if I understand well

It should be the reverse of Subscribe IMHO … See how it’s done on https://www.dismoi.io/eclaireurs/

**Comment by prk-dismoi** [comment]
ok we are clear on this now. I will provide a UI design to signpost relayed contributions.

**Comment by felix-lambert** [comment]
@JalilArfaoui are you available for a meeting to discuss about this?

**Comment by JalilArfaoui** [comment]
Sure !

**Comment by JalilArfaoui** [comment]
@felix-lambert yep !

**Comment by prk-dismoi** [comment]
renamed this issue > "display relayed contributions" if thats ok?

**Comment by felix-lambert** [comment]
Yes no problem

**Comment by prk-dismoi** [comment]
> ok we are clear on this now. I will provide a UI design to signpost relayed contributions.

@felix-lambert @JalilArfaoui, relayed contribution UI now on Zeplin (Android, 360x640):

web link > https://zpl.io/VY0AeXJ
app link > zpl://screen?sid=60f1b34c0d91690c36c265d6&pid=60e6d676382a1c713e9ba58e

**Comment by felix-lambert** [comment]
@prk-dismoi is this issue MVP? I wonder if we should not focus on making the app stable rather than adding small features?

**Comment by prk-dismoi** [comment]
Yep stability tickets take precedence. However experiencing it on my device right now, when you see a contribution from someone you know you didn't subscribe to makes you think the app is broken. Whatever we do, we should be address it quickly enough.
